radicle-node/src/service.rs

```diff
@@ -266,6 +272,16 @@ where
         self.reactor.next()
     }
 
+    /// Track a node.
+    /// Returns whether or not the tracking policy was added or updated.
+    pub fn track_node(
+        &mut self,
+        id: &NodeId,
+        alias: Option<&str>,
+    ) -> Result<bool, tracking::Error> {
+        self.tracking.track_node(id, alias)
+    }
```

Comment:
What's this function for? It doesn't seem to do anything but borrow all of `&self` instead of just borrowing `self.tracking`

Reply:
My intention was to be consistent with the existing [`track_repo` method](https://github.com/radicle-dev/heartwood/blob/f8aa1daedb391069847c66e14b2da6cc2e5669ae/radicle-node/src/service.rs#L271). 

I will change it to get a `&mut tracking` only. I'm okay with either way.